perun-scim/src/main/java/cz/metacentrum/perun/scim/api/entities/UserSCIM.java

```diff
@@ -0,0 +1,131 @@
+package cz.metacentrum.perun.scim.api.entities;
+
+
+import org.codehaus.jackson.annotate.JsonProperty;
+
+import java.util.List;
+
+import org.codehaus.jackson.map.annotate.JsonSerialize;
+
+/**
+ * User resource type for SCIM protocol.
+ *
+ * @author Sona Mastrakova <sona.mastrakova@gmail.com>
+ * @date 08.10.2016
+ */
+
+@JsonSerialize(include = JsonSerialize.Inclusion.NON_NULL)
+public class UserSCIM extends Resource {
+
+	@JsonProperty
+	private List<String> schemas;
+
+	@JsonProperty
+	private String userName;
+
+	@JsonProperty
+	private String name;
+
+	@JsonProperty
+	private String displayName;
+
+	@JsonProperty
+	private List<EmailSCIM> emails;
+
+	public UserSCIM(Long id, Long externalId, Meta meta, List<String> schemas, String userName, String name, String displayName, List<EmailSCIM> emails) {
+		super(id, externalId, meta);
+		this.schemas = schemas;
+		this.userName = userName;
+		this.name = name;
+		this.displayName = displayName;
+		this.emails = emails;
+	}
+
+	public UserSCIM(Long id, Long externalId, Meta meta) {
+		super(id, externalId, meta);
+	}
+
+	public UserSCIM() {
+	}
+
+	public List<String> getSchemas() {
```

Comment:
Getters which return collections should do that as unmodifiable list or immutable collection.